.travis.yml

```diff
@@ -16,4 +16,4 @@ matrix:
     script:
       - make dev
     after_success:
-      - make travis_coverage
+      - travis_wait make travis_coverage
```

Comment:
Yes, though highly unlikely. There must be something wrong in our tests, if it happened.